pkg/scaffold/external_templates_test.go

```diff
@@ -266,8 +264,9 @@ func TestCopyTemplateFiles(t *testing.T) {
 		}
 		stack2Configs[stack.Name] = configs
 	}
-	// project name
-	projectName := localTemplate.ProjectName
-	err = CopyTemplateFiles(localRoot, tmp, true, projectName, projectConfigs, stack2Configs)
+	err = CopyTemplateFiles(localTemplate.Dir, tmp, true, localTemplate.ProjectName, projectConfigs, stack2Configs)
+	if err != nil {
+		fmt.Printf("err: %v", err)
```

Comment:
It is recommended not to use `fmt.Printf` output error directly in a unit test

Reply:
sorry...this one is used to debug error...